src/components/story/sections/StoryCardListSection.tsx

```
@@ -44,12 +44,21 @@ function StoryCardListSection() {
     hasNextPage,
     fetchNextPage,
     isFetchingNextPage,
   );
-  const storyList = data?.pages.flatMap(p => p.results ?? []);
+
+  const storyList = useMemo(() => {
+    const allStories = data?.pages.flatMap(p => p.results ?? []);
+
+    if (!allStories || allStories.length === 0) return [];
+
+    const validStories = allStories.filter(story => story && story.storyId);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)